### ISSUE TITLE
Add support to Lenovo Tab Pen Plus

### ISSUE THREAD

**Issue by bjornasm** [saber-notes/saber#1361]
### Before submitting an issue, please check the following

- [X] I have searched for similar issues (both open and closed) and cannot find a duplicate
- [X] I agree to follow the [Code of Conduct](https://github.com/saber-notes/saber/blob/main/CODE_OF_CONDUCT.md)

### Motivation

I seems like Lenovo Tab Pen Plus is not supported by this app, as it apparently needs active support.

### Solution

It seems like lenovo suggests that each software have to add support for the different pen versions, and Lenovo Tab Pen Plus seems not to be supported here as it doesn't work in this app using my tablet. (It works in some other apps).

### Saber version

v0.24.7 Google Play (24070)

### Device

- Device: Lenovo Tap M11
- OS: Lenovo ZUI 16.0.043 Android version 14

**Comment by erik-404err** [comment]
It worked for me out of the box:

**Saber Version**

v0.24.7 F-Droid (24070)

**Device**

- Device: Lenovo Tab P12
- OS: Lenovo ZUI 16.0.081 Android 14
- Stylus: Lenovo Tab Pen Plus
- Stylus Firmware: V1.14

I hope this helps with troubleshooting/implementation

**Comment by bjornasm** [comment]
Thank you for your input @erik-404err . I am sorry but it seems like I made this issue a bit to hastly. Can you also press the side button on the pen to erase what you have wrote with the pen?

**Comment by erik-404err** [comment]
No, the button for changing pen/erase does also not work for me. For the time being you could use "KeyMapper" from F-Droid and the "Remote Control" Mode of the Lenovo Pen. The eraser is toggeld with "Ctrl + E".
I agree that this would definitely be a great enhancement to the project, if build in.

**Comment by bjornasm** [comment]
Thank you for that suggestion @erik-404err !
